.github/workflows/rtlmeter.yml

```diff
@@ -357,7 +350,7 @@ jobs:
       - name: Create issue
         env:
           GH_TOKEN: ${{ steps.generate-token.outputs.token }}
-        run: |
+        run: |-
```

Comment:
Would have expected this to be breaking as Yaml `|-` strips the `\n` between lines so they would be fed to the shell as one line, but the run seems to be ok with it.